Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -44,6 +52,71 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
             });
 
             this._publishEvents();
+
+            this.on('changeTab', this._updateVisibleMethod);
+        },
+
+        /**
+         * Updates the tab to show the correct tab depending on the visible
+         * method
+         *
+         * @method _uiUpdateTab
+         * @protected
+         */
+        _uiUpdateTab: function () {
+            var container = this.get('container'),
+                htmlId = '#' + this._visibleMethodView.getHTMLIdentifier();
+
+            this._selectTab(
+                this._getTabLabel(container.one('[href="' + htmlId + '"]')),
+                htmlId,
+                container
+            );
+        },
+
+        /**
+         * Updates the visible flag of the method views depending on the value
```

Comment:
The description is unclear to me. What the `method` means in the descripton above?